host-spec/ae-hostapi.tm

```diff
@@ -1304,34 +1304,10 @@
     <item><strong|><verbatim|data>: a pointer-size as defined in Definition
     <reference|defn-runtime-pointer> indicating the data to be hashed.
 
-    <item><verbatim|return>: a regular pointer to the buffer containing the
+    <item><verbatim|return>: a reglar pointer to the buffer containing the
```

Comment:
Looks like you undid your some upstream work here. But should be easy enough to just reapply #362.